docs/GettingStarted.rst

```diff
@@ -300,6 +300,485 @@ redirected to the :ref:`Social Intranet homepage <PLFHomepage>`. Now, you can st
        - \*:/developers
 
 	* If the server stops before your account setup data is submitted, the Account Setup screen will appear at your next startup.
+
+  .. _DWHomepage:
+
+==========================
+Digital Workplace Homepage
```

Comment:
OK to have section top describe this page, but IMO we need a chapter to talk about the digital workplace portal as a whole and a descriprion of its navigations and main pages and apps